cms/search/tests/test_api_views.py

```diff
@@ -37,7 +37,7 @@ def setUpTestData(cls):
             InformationPageFactory(),
             MethodologyPageFactory(),
             ReleaseCalendarPageFactory(),
-            StatisticalArticlePageFactory(),
+            StatisticalArticlePageFactory(news_headline=""),
```

Comment:
As discussed, let's quickly check with Paula/Amelia to confirm this is the expected behaviour. Do they want to use the news headline since it is what is displayed on the topic page or just the full title with series.